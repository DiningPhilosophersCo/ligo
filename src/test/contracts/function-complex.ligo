// Test a PascaLIGO function with more complex logic than function.ligo

function main (const i : int) : int is
  block {
    var j : int := 0;
    var k : int := 1;
    j := k + i;
    k := i + j
  } with k + j
