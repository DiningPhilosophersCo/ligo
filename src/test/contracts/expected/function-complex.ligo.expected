function main (const i : int) : int is
block {
  var j : int := 0;
  var k : int := 1;
  j := k + i;
  k := i + j
} with k + j
